MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -177,6 +177,11 @@ public ServerConfiguration()
     /// <value>The library update duration.</value>
     public int LibraryUpdateDuration { get; set; } = 30;
 
+    /// <summary>
+    /// Gets or sets the amount of items to cache.
+    /// </summary>
+    public int LibraryCacheSize { get; set; } = 10_000;
```

Comment:
I have 161k in my "TypedBaseItems" table, so I just picked 10k for now.

Reply:
Correct my assumption if wrong, but this gets used in the `.WithCapacity` in _LibraryManager.cs_ and in _UserLibraryManager.cs_ so we could have 10K of each, correct? Also assuming that the caches in _DirectoryService.cs_ are not capacity-constrained... how many will they cache?

Reply:
🤷 as many as it currently does. The only change there was to make the cache entries expire after 10 minutes of no access

My thought process is that the DirectoryService cache is hit the hardest during a library scan so increased memory is expected, but after scan it should drop back down

Reply:
If a cache isn't initialized with a size it is set to 128, which is much too small for our use case.